src/MultiFormGoals/resources/js/components/server-side-render-x/index.js

```diff
@@ -1,3 +1,14 @@
+/**
+ * NOTE: This is a fork of the original ServerSideRenderX component.
+ * You can find more about that project here: https://github.com/dgwyer/server-side-render-x
+ *
+ * Specifically, for GiveWP we have opted to remove the spinner/loading state that ServerSideRenderX uses entirely.
```

Comment:
@henryholtgeerts we may want to consider adding the spinner back in. Locally, it feels like the updates are un-responsive, which the spinner mitigates.